pkg/encryption/encryption.go

```diff
@@ -33,78 +33,86 @@ func Increment(nonce *storj.Nonce, amount int64) (truncated bool, err error) {
 }
 
 // Encrypt encrypts data with the given cipher, key and nonce
-func Encrypt(data []byte, cipher storj.Cipher, key *storj.Key, nonce *storj.Nonce) (cipherData []byte, err error) {
+func Encrypt(data []byte, cipher storj.CipherSuite, key *storj.Key, nonce *storj.Nonce) (cipherData []byte, err error) {
 	// Don't encrypt empty slice
 	if len(data) == 0 {
 		return []byte{}, nil
 	}
 
 	switch cipher {
-	case storj.Unencrypted:
+	case storj.EncUnspecified:
```

Comment:
This is not the correct behavior, it should return an error here.

Reply:
updated